frontend/public/components/sidebars/resource-quota-sidebar.jsx

```diff
@@ -0,0 +1,36 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { ResourceQuotaModel } from '../../models';
+import { referenceForModel } from '../../module/k8s';
+import { SampleYaml } from './resource-sidebar';
+
+const samples = [
+  {
+    highlightText: 'Specify',
+    header: 'namespace for the resource quota to apply to',
+    details: 'A namespace identifier can be used to limit the scope of where the resource quota is applied.',
```

Comment:
Isn't ResourceQuota always namespaced?

We are looking at adding ClusterResourceQuota in #458. I think we could add cluster-scoped examples there.

I think the samples we'd want to highlight are probably compute resources (request and limit), object counts, and storage classes. We might want an example with quota scopes as well.

Reply:
Resource quota are always namespaced.  Maybe I should have said 'a namespace identifier is used to define the scope of where the resource quota is applied.'

Reply:
Yeah, the way you've got it, it seems to imply the other examples aren't namespaced. I think I would change this one to be an example of compute resource quotas with different quota scopes (probably with comments in the YAML describing the scope)

Reply:
Agreed, I removed it and just created a compute example.